build_and_push.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+docker build -t geometalab/osmboundaries:latest .
```

Comment:
If you use `.` in a bash script, maybe `cd $(dirname $0)` first, so that the current working dir is the one this script is in. (Doesn't work if the script is sourced, or accessed via a symlink or hardlink in a different dir, but [that'd be difficult to solve](http://stackoverflow.com/questions/59895/can-a-bash-script-tell-what-directory-its-stored-in) and YAGNI.)